src/receivers/HTTPReceiver.ts

```diff
@@ -519,3 +547,31 @@ function parseBody(req: BufferedIncomingMessage) {
   }
   return JSON.parse(bodyAsString);
 }
+
+function prepareCustomRoutes(customRoutes: CustomRoute[]): CustomRoute[] {
+  const requiredKeys: (keyof CustomRoute)[] = ['path', 'method', 'callback'];
+  const missingKeys: (keyof CustomRoute)[] = [];
+
+  // Check for missing required keys
+  customRoutes.forEach((route) => {
+    requiredKeys.forEach((key) => {
+      if (route[key] === undefined && !missingKeys.includes(key)) {
+        missingKeys.push(key);
+      }
+    });
+  });
+
+  if (missingKeys.length > 0) {
+    const errorMsg = `One or more members of customRoutes are missing required keys: ${missingKeys.join(', ')}`;
+    throw new CustomRouteInitializationError(errorMsg);
```

Comment:
As codecov suggests, we can have a simple test covering this scenario

Reply:
Oops! Forgot to circle back and add this here, as well. Thanks!